docs/datadog_monitor.md

```diff
@@ -52,6 +52,18 @@ To deploy a `DatadogMonitor` with the Datadog Operator, use the [`datadog-operat
 
     For additional examples, see [examples/datadog-monitor](../examples/datadogmonitor). Note that only metric alerts, query alerts, and service checks are supported.
 
+   By default, the Operator only watches its own namespace, so it will manage any `DatadogAgent` and `DatadogMonitor` objects within its own namespace. Therefore, you should deploy your Datadog objects in the same namespace as the Operator. If you'd like to deploy your DatadogMonitors in different namespaces, then you will need to configure the Operator [`watchNamespaces`][8] section with those additional namespaces:
```

Comment:
```suggestion
   By default, the Operator only watches its own namespace, so it will manage any `DatadogMonitor` objects within its own namespace. Therefore, you should deploy your Datadog objects in the same namespace as the Operator. If you'd like to deploy your DatadogMonitors in different namespaces, then you will need to configure the Operator [`watchNamespaces`][8] section with those additional namespaces:
```

Since this doc is for DatadogMonitor, I think it's ok to remove this

Reply:
```suggestion
   By default, the Operator only watches its own namespace, so it will manage any `DatadogAgent` and `DatadogMonitor` objects within its own namespace. Therefore, you should deploy your Datadog objects in the same namespace as the Operator. If you'd like to deploy your DatadogMonitors in different namespaces, then you will need to configure the Operator [`watchNamespaces`][8] section of your Helm values file with those additional namespaces:
```